examples/reference-entity-model/gradle-dhs.properties

```diff
@@ -17,3 +17,6 @@ mlAppServicesPort=8010
 
 mlUsername=hub-developer
 mlPassword=password
+
+mlFlowOperatorRole=flowOperator
```

Comment:
Yuck - were there things breaking when using DHF in DHS because these weren't overridden? I guess that's because we still use this for some default permissions.

Reply:
yes, when we deploy users its failing bcs of this. I think we said flowDeveloper and operator will continue to  work in 5.2.x

Reply:
You should also add mlFlowDeveloperRole=flowDeveloper

Reply:
My bad. I missed it